core/primitives-core/src/config.rs

```diff
@@ -374,6 +374,9 @@ pub struct ExtCostsConfig {
     /// Trie iterator next key byte cost
     pub storage_iter_next_value_byte: Gas,
 
+    pub storage_block_hash_base: Gas,
+    pub storage_iter_next_block_hash: Gas,
```

Comment:
I am not super happy with the `storage` prefix here. The other parameters with that prefix are for host functions that also have that prefix.

Maybe something like:
- `block_hash_base`
- `block_hash_per_block` or `block_hash_next_block` or something along those lines